src/modules/PositionModule.cpp

```diff
@@ -204,6 +204,8 @@ int32_t PositionModule::runOnce()
     }
 
     meshtastic_NodeInfoLite *node = nodeDB.getMeshNode(nodeDB.getNodeNum());
+    if (node == nullptr)
+        return RUNONCE_INTERVAL;
```

Comment:
Would be way better to also log an error here (instead of silently ignoring the fact that this get function betrayed us).
==> LOG_ERROR("getMeshNode() returned nullptr!");

Imagine we'd have dozens of this kind of code lines in the software where severe problems are simply ignored, how you're gonna pinpoint any bug in the system?

Reply:
Good point